src/fhirtypes/metaDataTypes.ts

```diff
@@ -19,4 +25,5 @@ export type UsageContext = {
   valueCodeableConcept?: CodeableConcept;
   valueQuantity?: Quantity;
   valueRange?: Range;
+  valueReference?: any;
```

Comment:
I'd suggest you go ahead and create the `Reference` type and put it in the `dataType.ts` file:
http://hl7.org/fhir/R4/references.html#Reference